lncrawl/templates/novelupdates.py

```diff
@@ -138,6 +150,8 @@ def parse_chapter_item(self, tag: Tag, id: int) -> Chapter:
     def download_chapter_body_in_scraper(self, chapter: Chapter) -> None:
         response = self.get_response(chapter.url, allow_redirects=True)
         logger.info("%s => %s", chapter.url, response.url)
+        if "re-library" in response.url and "translations" not in response.url:
+            response.url = self.relibrary_url_transformation(response, chapter)
```

Comment:
I would prefer this logic to be in the `parse_chapter_body` section. 

You can use the `chapter.url` and `text` parameters of the  `parse_chapter_body` for relibrary_url_transformation.